ami/ml/orchestration/test_cleanup.py

```
@@ -0,0 +1,211 @@
+
+        # Verify NATS stream and consumer are deleted
+        async def check_nats_resources():
+            async with TaskQueueManager() as manager:
+                stream_name = manager._get_stream_name(job_id)
+                consumer_name = manager._get_consumer_name(job_id)
+
+                # Try to get stream info - should fail if deleted
+                stream_exists = True
+                try:
+                    await manager.js.stream_info(stream_name)
+                except Exception:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)